data-tck/src/main/java/io/micronaut/data/tck/repositories/BookRepository.java

```diff
@@ -81,8 +81,11 @@ public BookRepository(AuthorRepository authorRepository) {
     @Join("author")
     public abstract Book findByTitle(String title);
 
+    @Join(value = "author.books", type = Join.Type.LEFT_FETCH)
     public abstract Author findAuthorById(@Id Long id);
```

Comment:
Can you also please check the method with a default join type

Reply:
Added new method getAuthorById with default join, works the same.